tools/ai2_common.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+# 0-*- Mode: Python; tab-width: 4; indent-tabs-mode: nil; coding: utf-8; -*-
+
+import sys
+import os.path
+import spacy
+import itertools
+import glob
+import re
+from sys import path as sys_path
+
+# this seems to be necessary for annotations to find its config
+sys_path.append(os.path.join(os.path.dirname(__file__), '..'))
+try:
+    import annotation
+except ImportError:
+    # Guessing that we might be in the brat tools/ directory ...
+    sys_path.append(os.path.join(os.path.dirname(__file__), '../server/src'))
+    import annotation
+
+NLP = spacy.load("en")
```

Comment:
This may be problematic if the module is imported from another. Consider passing this to the class constructor instead of making it global.